scripts/build-controller.sh

```diff
@@ -141,7 +143,7 @@ config_output_dir="$SERVICE_CONTROLLER_SOURCE_PATH/config/"
 pushd $SERVICE_CONTROLLER_SOURCE_PATH/apis/$ACK_GENERATE_API_VERSION 1>/dev/null
 
 echo "Generating deepcopy code for $SERVICE"
-controller-gen object:headerFile=$TEMPLATES_DIR/boilerplate.txt paths=./...
+controller-gen object:headerFile=$TEMPLATE_DIRS/boilerplate.txt paths=./...
```

Comment:
I know this won't work for multiple paths but wanted to raise the PR and discuss the question of the two scripts first